test/e2e-tests.sh

```diff
@@ -40,6 +40,21 @@ header "Setting up environment"
 install_pipeline_crd
 
 failed=0
+function alpha_gate() {
+  local gate="$1"
+  if [ "$gate" != "alpha" ] && [ "$gate" != "stable" ] && [ "$gate" != "beta" ] ; then
+    printf "Invalid gate %s\n" ${gate}
+    exit 255
+  fi
+  if [ "$gate" == "alpha" ] ; then
+    DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+    printf "Setting up environement for alpha features"
+    install_spire
+    patch_pipline_spire
+    kubectl patch configmap config-spire -n tekton-pipelines -f "$DIR"/testdata/spire/config-spire.yaml
+    failed=0
+  fi
+}
```

Comment:
why do we create a new func here, can it be merged into set_feature_gate() ?

Reply:
This is just to keep the size of the set_feature_gate small and also keep this only for SPIRE initialization.
We could merge it  into set_feature_gate.